### PR TITLE
feat: improve app update experience when running on electron

### LINKED ISSUE #2876: Improve app update experience
## Abstract

Improve how updates are presented to the user once (s)he has logged in

## Motivation

After a new release is published the user is prompted in the login screen  with a `download` button. If the user doesn't upgrade s(he) can see a red arrow in the top bar but this opens the release notes instead of the popup with the option to download the update

## Specification

If the user clicks on the red (up pointing) arrow rotki should present a bubble with the option to download the update (as we do in the login screen) but would be also interesting to also link the github release at github

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #2876

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Improve app update experience when running on electron.

### CODE REVIEW

**Comment by yabirgb** [comment]
How could I test it?

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2876` App update experience is improved when running on electron.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: how is the experience updated? It's not clear from the changelog. Please explain in a few more words.

**Comment by kelsos** [comment]
Code is a pass from me @yabirgb will you check the functionality?

**Comment by yabirgb** [comment]
> Code is a pass from me @yabirgb will you check the functionality?

Yes, one sec. Will try now

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**